src/nvim/api/buffer.c

```diff
@@ -972,7 +972,7 @@ Integer nvim_buf_add_highlight(Buffer buffer,
 /// To clear the namespace in the entire buffer, pass in 0 and -1 to
 /// line_start and line_end respectively.
 ///
-/// @param buffer     Buffer handle
+/// @param buffer     Buffer handle, or 0 for current buffer
```

Comment:
Why change this on one function only?

Reply:
I'll change it for all cases. I was trying to use this one for a script and noticed it wasn't specified.